### PR TITLE
Commentlist feature base

### PR DESCRIPTION
**Pull request by caffreydev**

### CODE REVIEW

File: src/components/Article.jsx [hunk 1]
```diff
@@ -33,23 +35,29 @@ const {articleId} = useParams();
...
 
 return (
+    <>
     <div className="article-full">
         <h2>{result.title}</h2>
         <p>Penned by {result.author}</p>
         <p>Topic of {result.topic}</p>
-        <p>{dateFormatter(result.created_at)}</p>
+        <p>Published on {dateFormatter(result.created_at)}</p>
         <h3>Article Text:</h3>
```
Comment (NRMSMITH) [inline src/components/Article.jsx:55]: I'm not sure that you need this h3 - it'll be intuitive without

File: src/components/CommentList.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            setError(true)
+        })
+    }, [])
+
+    if (error) {
+        return <p className="warning-text">{errMsg}</p>
+    } else if (loading) {
+        return <p>Please wait while comments load</p>
+    } 
+
```
Comment (NRMSMITH) [inline src/components/CommentList.jsx:37]: What should appear if an article has no comments?

**Review by NRMSMITH — CHANGES_REQUESTED** [review]
Have a think about what should happen if the return value of the api call is an empty array :)

File: src/components/CommentList.jsx (hunk 2 above)
Comment (caffreydev) [reply src/components/CommentList.jsx:37]: Good point - I should invite the user to be the first to comment of course! now added

File: src/components/Article.jsx (hunk 1 above)
Comment (caffreydev) [reply src/components/Article.jsx:55]: agree, it's gone

**Pull request merged**